polaris-react/src/components/AppProvider/global.css

```diff
@@ -110,11 +110,9 @@ html {
   system font and then define font-families and rem-based
   font-sizes on descendant elements:
 */
-@supports (font: -apple-system-body) {
-  @media (--p-breakpoints-sm-down) {
-    html {
-      font: -apple-system-body;
-    }
+@supports (font: -apple-system-body) and (-webkit-touch-callout: none) {
```

Comment:
Just wanted to confirm that this fix is only targeting Safari on iOS? We don't need to fix this on other browsers that an iPad could be using?
![Screenshot 2024-04-23 at 10 51 00 AM](https://github.com/Shopify/polaris/assets/21976492/c905caf1-b893-41fe-b867-423d7632a428)

Reply:
I'll test to confirm it but I think all browsers on iOS use safari under the hood. 
That is also the case for the webviews in the app.